src/Microsoft.Diagnostic.Repl/Command/CommandProcessor.cs

```diff
@@ -112,11 +120,13 @@ private void BuildCommands(CommandLineBuilder rootBuilder, IEnumerable<Assembly>
                                 if (argument != null) {
```

Comment:
Good question.  The UseHelp/UseHelpBuilder in the constructor seems to be enough to handle the "command -?|--help" case.  I'm not sure even if this UseHelp is needed anymore. I'll investigate.

Reply:
`UseHelp` configures the parser and middleware to understand various help tokens from the command line input, short circuit rather than invoking your handler, and return contextual help output instead.

`UseHelpBuilder` specifies which `IHelpBuilder` implementation will be used to write the output.

Hope that's useful.

Reply:
Removed this builder/builder.UseHelp() call. The main one when I build the parser is enough.